dim_checks/mozdata/fenix/active_users_aggregates/dim_checks.yaml

```diff
@@ -0,0 +1,41 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  slack_alerts:
+    enabled: true
```

Comment:
I received a request from Data Science to only send alerts when the checks fail, which I think makes sense. Then we wouldn't fail to check an actual error when it happens, based on the assumption that it might be a success alert. What do you think?

Reply:
cc: @gkabbz are you okay for us to switch over to notify on failures only or should we continue sending out a slack on each run?

Reply:
This was implemented by  publishing all results but notifying @channel only for failures.

Reply:
yes, this PR introduced this change:
 https://github.com/mozilla/dim/pull/87